### ISSUE TITLE
Certificates cannot be generated without updating the web.config key AllowCAGen

### ISSUE THREAD

**Issue by d0lb33** [theopenem/Toems#10]
Make this more obvious in the UI, as all the Frontend says is "could not generate certs" and i had no idea why until I followed the code to the certificate generation class. 

It doesn't seem to be logging anywhere I can find. Maybe this needs to be a logging.info call?
